packages/backend/src/core/balances/BalanceUpdater.ts

```diff
@@ -39,6 +36,10 @@ export class BalanceUpdater {
   ) {
     this.logger = this.logger.for(this)
     this.configHash = getBalanceConfigHash(projects)
+    this.taskQueue = new TaskQueue(
+      this.update.bind(this),
```

Comment:
can we normalize to () => this.update()

Reply:
I will do this, but in a separate PR - will rename updaters to counters - lets RIP this PR.